chips/nrf52/src/ieee802154_radio.rs

```diff
@@ -4,10 +4,11 @@
 
 //! IEEE 802.15.4 radio driver for nRF52
 
+use crate::timer::TimerAlarm;
 use core::cell::Cell;
 use core::convert::TryFrom;
 use kernel;
-use kernel::hil::radio::{self, PowerClient};
+use kernel::hil::radio::{self, PowerClient, RadioData};
 use kernel::hil::time::{Alarm, AlarmClient};
```

Comment:
E.g., import `Time` here to be able to use `.now()` instead of the new `get_current_time()`

```suggestion
use kernel::hil::time::{Alarm, AlarmClient, Time};
```